skrebate/surfstar.py

```diff
@@ -66,28 +69,29 @@ def _find_neighbors(self, inst, avg_dist):
 
         return np.array(NN_near, dtype=np.int32), np.array(NN_far, dtype=np.int32)
 
+
     def _run_algorithm(self):
+        """ Runs nearest neighbor (NN) identification and feature scoring to yield SURF* scores. """
         sm = cnt = 0
         for i in range(self._datalen):
             sm += sum(self._distance_array[i])
             cnt += len(self._distance_array[i])
         avg_dist = sm / float(cnt)
 
-        attr = self._get_attribute_info()
         nan_entries = np.isnan(self._X)
 
         NNlist = [self._find_neighbors(datalen, avg_dist) for datalen in range(self._datalen)]
         NN_near_list = [i[0] for i in NNlist]
         NN_far_list = [i[1] for i in NNlist]
 
-        if self.n_jobs != 1:
+        if self.n_jobs != 1: #Parallelization
```

Comment:
I think we could remove `if self.n_jobs != 1` in **all** rebate-based algorithms for reducing redundant codes